__tests__/reactTags.test.js

```diff
@@ -426,9 +426,15 @@ describe('Test ReactTags', () => {
         defaults.suggestions
       );
 
-      $input.simulate('change', { target: { value: 'ea' } });
+      $input.simulate('change', { target: { value: 'or' } });
       expect(ReactTagsInstance.state.suggestions).to.have.members([]);
 
+      $input.simulate('change', { target: { value: 'ea' } });
```

Comment:
Added 'ea' test case for handle partial search behavior. 
'or' test case still here for handle  empty search behavior.